docs/tutorials/chatbot.md

```diff
@@ -0,0 +1,571 @@
+# Covid-19 Chatbot Tutorial
+
+
+We will use the {ref}`hello world chatbot <chatbot-helloworld>` for this tutorial. You can find the complete code [here](https://github.com/jina-ai/jina/tree/master/jina/helloworld/chatbot) and we will go step by step.
+
+At the end of this tutorial, you will have your own chatbot. You will use text as an input and get a text result as output. For this example, we will use a [covid dataset](https://www.kaggle.com/xhlulu/covidqa). You will understand how every part of this example works and how you can create new apps with different datasets on your own.
```

Comment:
Let's use this dataset through huggingface datasets: https://huggingface.co/datasets/covid_qa_castorini

It's 100x easier to work with it, we can remove all the verbose code that deals with downloading data

Reply:
this dataset is not compatible

Reply:
I see. Let me see what could be done here.